tools/cpm_tpm_rpk/cpm_tpm_rpk.xml

```diff
@@ -0,0 +1,100 @@
+		<param name="input_sep" type="select" label="Input file separator">
+			<option value="tab" selected="true">Tabs</option>
+			<option value=",">Comma</option>
+			<option value="\ ">Whitespace</option>
+			<option value=".">Dots</option>
+                        <option value="_">Underscores</option>
+                        <option value="-">Dashes</option>
+		</param>
+		<param name="input_header" type="select" label="Consider first line of input file as header?">
+			<option value="TRUE" selected="true">Yes</option>
+			<option value="FALSE">No</option>
+		</param>
+                <param name="type_transfo" type="select" label="Type of transformation">
+      	                <option value="cpm" selected="true">CPM</option>
+               	        <option value="tpm">TPM</option>
+               	        <option value="rpk">RPK</option>
+               	</param>
+		<conditional name="tpm_rpk">
+			<param name="type_selector" type="boolean" truevalue="Yes" falsevalue="No" label="Do you want to transformed your data in TPM or RPK?"/>
+			<when value="Yes">
+				<param name="gene_file" type="data" format="txt" label="Gene length file"/> 
+				<param name="gene_sep" type="select" label="Gene length file separator">
+					<option value="tab" selected="true">Tabs</option>
+   	        			<option value=",">Commas</option>
+					<option value="\ ">Whitespaces</option>
+					<option value=".">Dots</option>
+					<option value="_">Underscores</option>
+					<option value="-">Dashes</option>
+				</param>
+				<param name="gene_header" type="select" label="Consider first line of gene length file as header ?">
+					<option value="TRUE" selected="true">Yes</option>
+					<option value="FALSE">No</option>
+				</param>
+			</when>
+			<when value="No">
+			</when>
+		</conditional>
+		<param name="log" type="select" label="Data should be log transformed ?">
+			<option value="FALSE" selected="true">No</option>
+			<option value="TRUE">Yes</option>
+		</param>
+	</inputs>
+	<outputs>
+                <data name="output" format="tabular" label="Transformation of raw counts" />
+	</outputs>
+    <tests>
+        <test>
+            <param name="input" value="counts.tab" ftype="tabular"/>
+            <param name="type_transfo" value="cpm"/>
+            <param name="type_selector" value="No"/>
+            <output name="output" file="cpm.tab" ftype="tabular"/>
+        </test>
+        <test>
+            <param name="input" value="counts.tab" ftype="tabular"/>
+            <param name="type_transfo" value="tpm"/>
+            <param name="type_selector" value="Yes"/>
+            <param name="gene_file" value="gene_length.tab" ftype="tabular"/>
+            <param name="gene_header" value="FALSE"/>
+            <output name="output" file="tpm.tab" ftype="tabular"/>
+        </test>
+    </tests>
+    <help>
+
+**What it does**
+
+cpm_tpm_rpk takes a table of raw read counts, each column corresponding to a sample count and 
```

Comment:
cpm_tpm_rpk takes a table of raw read counts, first column for gene names and other columns for sample counts,